docs/web_extension/liquid/getting_started.md

```diff
@@ -0,0 +1,75 @@
+# Getting Started
+
+Download the [Liquality Wallet](https://chrome.google.com/webstore/detail/liquality-wallet/kpfopkelmapcoipemfendmdcghnegimn?hl=en).
+
+## Connecting with the extension
+
+The extension injects the api into pages under `window.liquid`. 
+
+To check that the extension is running and accessible on your web application, log the `window.liquid` object.
+
+```js
+> console.log(window.liquid)
+{ enable: ƒ, request: ƒ }
+```
+
+As shown, the liquid interface exposes 2 very simple functions.
+
+## Enable
+
+Applications that need access to the wallet must first request that the extension is enabled. 
+
+```js
+> await window.liquid.enable()
+```
+
+This will prompt the user to allow access to the wallet for the application.
+
+![Wallet](../assets/enable.png)
+
+If the prompt is accepted, the first address of the user wallet will be returned. This can be used as a form of identity.
+
+```js
+[
+  {
+    "address": "el1qqd6q3nsyzj6pvdudqesjuw2v3ehxmu660y7ap7gp7zaaqn2df5qmdlmg6g6adt8r8sysxwn6s7m4hm2q83j0jha6v4hlq4755",
+    "blindingPrivateKey": "68ac4860741786b421a6ea91b3fdc77468b04ac014ec4f1e33018e9a7a9fdcc1",
+    "derivationPath": "84'/1'/0'/0/0",
+    "publicKey": "03269f25419716fa284c02f8186136f79b623a59edecd92c487bdfb1dfbdc6a4c8",
+    "index": 0
+  }
+]
+```
+
+The application can then make further requests to the wallet using `window.liquid.request()`
+
+## Request
+
+The `request` method is used to request information from the wallet (e.g. addresses, public keys) as well as prompt for signatures.
+
+For example: We can retrieve 10 addresses using the `wallet_getAddresses` method:
+
+```js
+> await liquid.request({ method: 'wallet_getAddresses', params: [0, 10] })
```

Comment:
What is the rationale behind using a JSON-RPC like model here? This interacts very poorly with strongly-typed languages. Personally, I would prefer APIs like:

```ts
await liquid.wallet.getAddresses(0, 10)
```

Reply:
This is more a broader topic/discussion to have about the entire CAL itself. About the rationale I'm not the right person to answer to this, as per other comment I followed what was already in place, I may be wrong, but mostly the idea as far I can tell was to follow Ethereum/web3.js model, which in turns calls a JSONRPC server most of the time. 

I agree also with this, I proposed offline indeed to have a typed binary serialization of messages (ie. Protocol Buffers) which would gives us typed specifications, self-generation of client/server stubs, backward&forward compatibility when changing/adding/removing message fields and also binary serialization ie. 3-5x faster that plain utf8 strings like JSONs.
Bonus point: self-generated API doc reference (since we are discussing in the documentation repo) :)

Reply:
This is more a broader topic/discussion to have about the entire CAL itself. About the rationale I'm not the right person to answer to this, as per other comment I followed what was already in place, I may be wrong, but mostly the idea as far I can tell was to follow Ethereum/web3.js model, which in turns calls a JSONRPC server most of the time. 

I agree also with this, I proposed offline indeed to have a typed binary serialization of messages (ie. Protocol Buffers) which would gives us typed specifications, self-generation of client/server stubs, backward&forward compatibility when changing/adding/removing message fields and also binary serialization ie. 3-5x faster that plain utf8 strings like JSONs.
Bonus point: self-generated API doc reference (since we are discussing in the documentation repo) :)

Reply:
This is more a broader topic/discussion to have about the entire CAL itself. About the rationale I'm not the right person to answer to this, as per other comment I followed what was already in place, I may be wrong, but mostly the idea as far I can tell was to follow Ethereum/web3.js model, which in turns calls a JSONRPC server most of the time. 

I agree also with this, I proposed offline indeed to have a typed binary serialization of messages (ie. Protocol Buffers) which would gives us typed specifications, self-generation of client/server stubs, backward&forward compatibility when changing/adding/removing message fields and also binary serialization ie. 3-5x faster that plain utf8 strings like JSONs.
Bonus point: self-generated API doc reference (since we are discussing in the documentation repo) :)